fem/gridfunc.hpp

```diff
@@ -365,28 +365,28 @@ class GridFunction : public Vector
                                  const IntegrationRule *irs[] = NULL) const;
 
    /** Compute the Lp error in each element of the mesh and store the results in
-       the GridFunction @a error. The result should be an L2 GridFunction of
-       order zero using map type VALUE. */
+       the Vector @a error. The result should be of length number of elements,
+       for example an L2 GridFunction of order zero using map type VALUE. */
    virtual void ComputeElementLpErrors(const double p, Coefficient &exsol,
-                                       GridFunction &error,
+                                       Vector &error,
```

Comment:
I think this is reasonable, and also backward compatible (a `GridFunction` is a `Vector` too).